tests/stdlib/tparsecfg.nim

```diff
@@ -78,3 +77,59 @@ name=hello
 name=lihf8515
 qq=10214028
 """
+
+block:
```

Comment:
add this diff:
```
diff --git a/tests/stdlib/tparsecfg.nim b/tests/stdlib/tparsecfg.nim
index bbf88ee03..dc1070f34 100644
--- a/tests/stdlib/tparsecfg.nim
+++ b/tests/stdlib/tparsecfg.nim
@@ -84,6 +84,7 @@ block:
   spaces in keys=allowed
   spaces in values=allowed as well
   spaces around the delimiter = obviously
+  quoted string = " spaces part of key  "
   you can also use : to delimit keys from values
   [All Values Are Strings]
   values like this: 19990429
@@ -99,6 +100,8 @@ block:
   [You can use comments]
   # like this
   ; or this
+  comment after line start = stops here # inside comment
+  comment after line start 2 = stops here 2 ; inside comment ; still inside comment
   # By default only in an empty line.
   # Inline comments can be harmful because they prevent users
   # from using the delimiting characters as parts of values.
@@ -116,6 +119,7 @@ block:
   doAssert dict.getSectionValue(section1, "spaces in keys") == "allowed"
   doAssert dict.getSectionValue(section1, "spaces in values") == "allowed as well"
   doAssert dict.getSectionValue(section1, "spaces around the delimiter") == "obviously"
+  doAssert dict.getSectionValue(section1, "quoted string") == " spaces part of key  "
   doAssert dict.getSectionValue(section1, "you can also use") == "to delimit keys from values"

   let section2 = "All Values Are Strings"
@@ -129,6 +133,10 @@ block:
   doAssert dict.getSectionValue(section3,
     "space around section name will be ignored", "not an empty value") == ""

+  let sectionComments = "You can use comments"
+  doAssert dict.getSectionValue(sectionComments, "comment after line start") == "stops here"
+  doAssert dict.getSectionValue(sectionComments, "comment after line start 2") == "stops here 2"
+
   let section4 = "Sections Can Be Indented"
   doAssert dict.getSectionValue(section4, "can_values_be_as_well") == "True"
   doAssert dict.getSectionValue(section4, "does_that_mean_anything_special") == "False"
```